### PR TITLE
Add chirp analysis function

### PR DESCRIPTION
**Pull request by t-b**
- [x] Commit cleanup
- [x] create bugfix PRs with general stuff
- [x] Add flowchart, including AB /BA ... logic
- [x] Add tests
- [x] Fix edge case which does not finish early: https://ai-mies.slack.com/archives/D1P2AN29F/p1591041938054200
- [x] Add dashboard support

Close https://github.com/AllenInstitute/MIES/issues/496.

### CODE REVIEW

File: Packages/MIES/MIES_AnalysisFunctions_PatchSeq.ipf [hunk 1]
```diff
@@ -2662,3 +2752,482 @@ static Function PSQ_ForceSetEvent(panelTitle, headstage)
...
+				WAVE/Z deltaI = GetLastSetting(numericalValues, passingDaScaleSweep, LABNOTEBOOK_USER_PREFIX + "Delta I", UNKNOWN_MODE)
+				WAVE/Z deltaV = GetLastSetting(numericalValues, passingDaScaleSweep, LABNOTEBOOK_USER_PREFIX + "Delta V", UNKNOWN_MODE)
+
+				if(!WaveExists(deltaI) || !WaveExists(deltaV))
+					printf "(%s): The Delta I/V labnotebook entries could not be found.\r", panelTitle
+					ControlWindowToFront()
+					return 1
+				endif
+
+				resistance = deltaV[s.headstage] / deltaI[s.headstage]
```
Comment (t-b) [inline Packages/MIES/MIES_AnalysisFunctions_PatchSeq.ipf:2946]: "empty comment"

**Comment by t-b** [comment]
Very early version is ready.

**Comment by t-b** [comment]
@timjarsky No need to test/review. This will be done by IVSCC once they start again.

**Comment by t-b** [comment]
I fixed the CI failures.
